### ISSUE TITLE
FormField component is not fully WCAG compliant (property "required")

### ISSUE THREAD

**Issue by vldmr1986** [grommet/grommet#6046]
I use AxeDev Tools chrome plugin for  WCAG compliance (https://design-system.hpe.design/foundation/accessibility#automated-testing-tools )
I see warnings for the following component:
                  <FormField
                        required
                        label={getLocalizedString('login.form.fields.password')}
                        htmlFor="password-login"
                        name="password"
                    >
                        <TextInput
                            id="password-login"
                            name="password"
                            placeholder={getLocalizedString('login.form.placeholders.password')}
                            type="password"
                        />
                    </FormField>

Issue Description
Ensures ARIA attributes are allowed for an element's role

Element location
label[for="password-login"] > .fxbXHC[aria-label="required"]
Element source
<span aria-label="required" class="StyledText-sc-1sadyjn-0 fxbXHC">*</span>
To solve this issue, you need to:
**Fix the following:**
aria-label attribute is not well supported on a span with no valid role attribute.

**Comment by avantika-codes** [comment]
Hello @jcfilben and @vldmr1986, I would like to work on this issue, could you please assign this to me?, meanwhile I am starting to work on the fixes

**Comment by jcfilben** [comment]
Hi @avantika-codes we usually don't assign issue but feel free to work on this and put up a pull request

**Comment by avantika-codes** [comment]
I am unable to test this issue at my end, as the Axe devtools is a paid service. Could you please help me out with some more details here?

**Comment by jcfilben** [comment]
Hey @avantika-codes there is a free version of axe dev tools but I just tested this in codesandbox and I'm not seeing axeDev tools flagging this issue anymore.
Here is the codesandbox I tested with: https://codesandbox.io/p/sandbox/grommet-v2-template-forked-gjys8f
I'm going to mark this issue as closed.
